app/homepage.py

```diff
@@ -26,24 +26,20 @@
 # Importing utils fetched API data
 from utils import build_live_alerts_metadata
 
-
 # ------------------------------------------------------------------------------
 # Before moving to the app layout
 
-#Fetching reusable alert metadata
-alert_metadata = build_live_alerts_metadata()
-frame_url = alert_metadata["media_url"]
-
 
-# This function creates a radio button in order to simulate an alert event that will be later caught by the API
+# This function creates a radio button for debug purposes, it simulates an alert event that will be
+# ater caught by the API
```

Comment:
Small typo in the comment and maybe we don't need the "that will be later caught by the API" anymore?

Reply:
Done